tests/posix/threads_base/src/pthread.c

```diff
@@ -499,7 +496,7 @@ ZTEST(pthread, test_sched_setscheduler)
 	zassert_true((rc == -1 && err == ENOSYS));
 }
 
-ZTEST(pthread, test_sched_rr_get_interval)
+ZTEST(posix_threads_base, test_sched_rr_get_interval)
```

Comment:
This test belongs in the `xsi_realtime` testsuite with the `_POSIX_PRIORITY_SCHEDULING` Option.